packages/amplify_core/lib/src/types/auth/attribute/auth_user_attribute_key.dart

```diff
@@ -16,15 +16,18 @@
 import 'package:amplify_core/amplify_core.dart';
 import 'package:meta/meta.dart';
 
-/// {@template amplify_core.user_attribute_key}
+@Deprecated('Use AuthUserAttributeKey instead')
```

Comment:
Should we add a TODO to remove this before GA?

Reply:
I can. I think all `Deprecated` tags should be removed for GA, so I've kind of been using them for that.